src/ast/mod.rs

```diff
@@ -824,6 +846,36 @@ impl fmt::Display for Statement {
                 }
                 Ok(())
             }
+            Statement::Deallocate { name, all, prepare } => {
+                f.write_str("DEALLOCATE ")?;
+                if *prepare {
+                    f.write_str("PREPARE ")?;
+                }
+                if *all {
+                    f.write_str("ALL")
+                } else {
+                    write!(f, "{}", name.as_ref().unwrap())
+                }
+            }
+            Statement::Execute { name, parameters } => {
+                write!(f, "EXECUTE {}", name)?;
+                if !parameters.is_empty() {
+                    write!(f, "({})", display_comma_separated(parameters))?;
+                }
+                Ok(())
+            }
+            Statement::Prepare {
+                name,
+                data_types,
+                statement,
+            } => {
+                write!(f, "PREPARE {} ", name)?;
+                if !data_types.is_empty() {
+                    write!(f, "({}) ", display_comma_separated(data_types))?;
+                }
+
```

Comment:
Again, I know we're inconsistent about this, but it seems that we usually don't add blank lines in situations like this.